### PR TITLE
CTS v3 query traces implemented

### LINKED ISSUE #486: trace management ctsv3
_Originally posted by @RusselSand in https://github.com/opentelekomcloud/python-otcextensions/issues/414#issuecomment-2460073611_

### PR DESCRIPTION
**Pull request by RusselSand**

### CODE REVIEW

**Review by vineet-pruthi — APPROVED** [review]

File: otcextensions/tests/unit/sdk/ctsv3/v3/test_traces.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    "record_time": 1481066128032,
+    "trace_id": "e001ccb9-bc09-11e6-b00b-4b2a61338db6"
+}
+
+
+class TestTrace(base.TestCase):
+    def test_basic(self):
+        sot = trace.Trace()
+        self.assertEqual('/traces', sot.base_path)
+        self.assertTrue(sot.allow_list)
```
Comment (vineet-pruthi) [inline otcextensions/tests/unit/sdk/ctsv3/v3/test_traces.py:47]: please also add  _query_mapping  params assertion to this test.

File: otcextensions/tests/unit/sdk/ctsv3/v3/test_traces.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+    def test_make_it(self):
+        sot = trace.Trace(**EXAMPLE)
+        self.assertEqual(EXAMPLE['time'], sot.time)
+        self.assertEqual(EXAMPLE['user']['name'], sot.user.name)
```
Comment (vineet-pruthi) [inline otcextensions/tests/unit/sdk/ctsv3/v3/test_traces.py:52]: please assert all attributes in the EXAMPLE.

File: otcextensions/sdk/ctsv3/v3/trace.py [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    resources_key = 'traces'
+
+    allow_list = True
+
+    _query_mapping = resource.QueryParameters(
+        'trace_type', 'limit', 'from', 'next', 'to', 'tracker_name',
+        'service_type', 'user', 'resource_id', 'resource_name',
+        'resource_type', 'trace_id', 'trace_name', 'trace_rating')
+
+    resource_id = resource.Body('resource_id')
```
Comment (vineet-pruthi) [inline otcextensions/sdk/ctsv3/v3/trace.py:43]: missing doc on the attributes

**Comment by anton-sidelnikov** [comment]
regate

**Pull request merged**